src/sort/simple-sort-directive.js

```diff
@@ -154,6 +154,26 @@ angular.module('patternfly.sort').directive('pfSimpleSort', function () {
     },
     templateUrl: 'sort/simple-sort.html',
     controller: function ($scope) {
+      var setSortDirectionTooltip = function () {
+        var tooltip = '';
+
+        if ($scope.config.currentField.sortType === 'numeric') {
+          if ($scope.config.isAscending) {
+            tooltip = 'Sort numerically, from least to greatest';
```

Comment:
Please pull the tooltips from the sort icons themselves, and only have tips on Sort by and Filter by.